components/org.wso2.carbon.micro.integrator.extensions/org.wso2.carbon.micro.integrator.management.apis/src/main/java/org/wso2/carbon/micro/integrator/management/apis/CarbonAppResource.java

```diff
@@ -162,24 +129,19 @@ private OMElement getCarbonAppByName(MessageContext messageContext, String carbo
         return null;
     }
 
-    private OMElement convertCarbonAppToOMElement(CarbonApplication carbonApp) throws XMLStreamException{
+    private JSONObject convertCarbonAppToOMElement(CarbonApplication carbonApp) {
 
         if (null == carbonApp) {
             return null;
         }
 
-        OMElement rootElement = AXIOMUtil.stringToOM(ROOT_ELEMENT_CARBON_APP);
-        OMElement nameElement = AXIOMUtil.stringToOM(NAME_ELEMENT);
-        OMElement versionElement = AXIOMUtil.stringToOM(VERSION_ELEMENT);
-        OMElement artifactsElement = AXIOMUtil.stringToOM(ARTIFACTS_ELEMENT);
+        JSONObject appObject = new JSONObject();
 
-        nameElement.setText(carbonApp.getAppName());
-        rootElement.addChild(nameElement);
+        appObject.put("name", carbonApp.getAppName());
```

Comment:
Shall we define variables for frequently used strings like **name**  , **versions** etc ?